docs/Agent/installation/ansible_based_installation.md

```diff
@@ -26,9 +22,9 @@ The same pbench-agent version must be installed on all the test systems that par
  export ANSIBLE_ROLES_PATH=$HOME/.ansible/collections/ansible_collections/pbench/agent/roles:$ANSIBLE_ROLES_PATH
  ```
 
-4. Create an inventory file (`~/.config/Inventory/myhosts.inv`) naming the hosts on which you wish to install pbench-agent and including information about where the config file and ssh key file can be found. Example inventory [file](assets/myhosts.inv).
+4. Create an inventory file (`~/.config/Inventory/myhosts.inv`) naming the hosts on which you wish to install pbench agent and including information about where the config file. Example inventory [file](assets/myhosts.inv).
```

Comment:
"where to find the config file", maybe? I didn't necessarily mean to drop the reference to the RSA key, because 0.72 still supports 0.69 and that's necessary. It's just that it's become more complicated because 0.72 supports both modes. Maybe "where to find the config file and, if you're planning to push performance data to a 0.69 Pbench Server, the server's private RSA key"?

Reply:
fixed